src/gateio_new_coins_announcements_bot/main.py

```diff
@@ -80,288 +80,330 @@ def buy():
-                            session[announcement_coin]['total_volume'] += (float(order[announcement_coin]['_amount']) * float(order[announcement_coin]['_price']))
-                            session[announcement_coin]['total_amount'] += float(order[announcement_coin]['_amount'])
-                            session[announcement_coin]['total_fees'] += float(order[announcement_coin]['_fee'])
-                            session[announcement_coin]['orders'].append(copy.deepcopy(order[announcement_coin]))
+                            order[announcement_coin]["_amount_filled"] = order[announcement_coin]["_amount"]
+                            session[announcement_coin]["total_volume"] += float(
+                                order[announcement_coin]["_amount"]
+                            ) * float(order[announcement_coin]["_price"])
+                            session[announcement_coin]["total_amount"] += float(order[announcement_coin]["_amount"])
+                            session[announcement_coin]["total_fees"] += float(order[announcement_coin]["_fee"])
+                            session[announcement_coin]["orders"].append(copy.deepcopy(order[announcement_coin]))
 
                             # update order to sum all amounts and all fees
                             # this will set up our sell order for sale of all filled buy orders
-                            tf = session[announcement_coin]['total_fees']
-                            ta = session[announcement_coin]['total_amount']
-                            order[announcement_coin]['_fee'] = f'{tf}'
-                            order[announcement_coin]['_amount'] = f'{ta}'
+                            tf = session[announcement_coin]["total_fees"]
+                            ta = session[announcement_coin]["total_amount"]
+                            order[announcement_coin]["_fee"] = f"{tf}"
+                            order[announcement_coin]["_amount"] = f"{ta}"
 
-                            store_order('order.json', order)
-                            store_order('session.json', session)
+                            store_order("order.json", order)
+                            store_order("session.json", session)
 
                             # We're done. Stop buying and finish up the selling.
                             globals.sell_ready.set()
                             globals.buy_ready.clear()
 
-                            logger.info(
-                                f'Order on {announcement_coin} closed',
-                                extra={'TELEGRAM': 'BUY_FILLED'})
+                            logger.info(f"Order on {announcement_coin} closed", extra={"TELEGRAM": "BUY_FILLED"})
                         else:
-                            if order_status == "cancelled" and float(order[announcement_coin]['_amount']) > float(order[announcement_coin]['_left']) and float(order[announcement_coin]['_left']) > 0:
```

Comment:
black reformatted some lines due to the line length limit of 120.
We should either do the right thing and reduce the amount of indentation for this method or maybe increase the limit to allow these long statements.

Reply:
- [ ] How should this get fixed?

Reply:
Happy to increase the limit here

Reply:
Won't it be more efficient to extract `float(order[announcement_coin]['_left'])` to a separate variable in order to make it more readable?
Usually, length of 120 chars considers as a proper one

Reply:
Sadly this is not the only case where the lines got split weirdly (see main.py). 
I would agree to refactor the code and try to condense it down and keep a max line length of 120.
However I don't wanna add even more complexity to this PR.

(Option 1) I would refactor these "ugly" lines in future PRs and for now leave it like this (which decreases readability but doesn't add more commits to this PR)
OR 
(Option 2) Increase the line length to something like 200 and make them readable again (in this PR) and later change it back to 120 and refactor the lines then (in individual PRs)

Reply:
We will go with Option 1 for now and work on it in future PRs, since this is only merged into develop